micro-ui/web/micro-ui-internals/packages/modules/core/src/components/TopBarSideBar/SideBar/EmployeeSideBar.js

```
@@ -94,61 +70,118 @@ const EmployeeSideBar = () => {
+    return configEmployeeSideBar;
+  };
+
+  const navigateToRespectiveURL = (history = {}, url = "") => {
+    if (url == "/") {
+      return;
+    } 
+    if (url?.indexOf(`/${window?.contextPath}`) === -1) {
+      const hostUrl = window.location.origin;
+      let updatedUrl=null;
+      if(isMultiRootTenant){
+        url=url.replace("/sandbox-ui/employee", `/sandbox-ui/${tenantId}/employee`);
```

Biome

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)